### PR TITLE
[MRG] EOG and ECG channels added to the bottom of ICA plot.

### PR DESCRIPTION
**Pull request by jaeilepp**
Proposed at https://github.com/mne-tools/mne-python/pull/2251.
screenshot from 2015-07-15 16 43 08

### CODE REVIEW

**Comment by dengemann** [comment]
looks clean to me! I see a bit of code reduplication. Maybe worth a small function. Other than that do you thinki we should add a test?

**Comment by agramfort** [comment]
is this a MRG PR?

File: mne/viz/ica.py [hunk 1]
```diff
@@ -626,9 +646,24 @@ def _plot_sources_epochs(ica, epochs, picks, exclude, start, stop, show,
...
+        ch_types = np.append(ch_types, 'eog')
+    for ecg_idx in ecg_chs:
+        c_names.append(epochs.ch_names[ecg_idx])
+        ch_types = np.append(ch_types, 'ecg')
+    extra_picks = np.append(eog_chs, ecg_chs).astype(int)
+    if len(extra_picks) > 0:
+        eog_ecg_data = np.concatenate(epochs.get_data()[:, extra_picks],
+                                      axis=1)
+        data = np.append(data, eog_ecg_data, axis=0)
+    scalings = {'misc': 5.0, 'eog': 150e-6, 'ecg': 5e-4}
```
Comment (dengemann) [inline mne/viz/ica.py]: use the defaults._handle_default

**Comment by jaeilepp** [comment]
Ready for merge?

**Comment by agramfort** [comment]
not yet... why is this not captured by tests?

```
In [2]: ica.plot_sources(epochs)
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-2-99fbf26e752b> in <module>()
----> 1 ica.plot_sources(epochs)

/Users/alex/work/src/mne-python/mne/preprocessing/ica.py in plot_sources(self, inst, picks, exclude, start, stop, title, show, block)
   1389         return plot_ica_sources(self, inst=inst, picks=picks, exclude=exclude,
   1390                                 title=title, start=start, stop=stop, show=show,
-> 1391                                 block=block)
   1392
   1393     def plot_scores(self, scores, exclude=None, axhline=None,

/Users/alex/work/src/mne-python/mne/viz/ica.py in plot_ica_sources(ica, inst, picks, exclude, start, stop, show, title, block)
    116         fig = _plot_sources_epochs(ica, inst, picks, exclude, start=start,
    117                                    stop=stop, show=show, title=title,
--> 118                                    block=block)
    119     elif isinstance(inst, Evoked):
    120         sources = ica.get_sources(inst)

/Users/alex/work/src/mne-python/mne/viz/ica.py in _plot_sources_epochs(ica, epochs, picks, exclude, start, stop, show, title, block)
    693                                n_epochs=n_epochs, scalings=scalings,
    694                                title=title, picks=picks,
--> 695                                order=['misc', 'eog', 'ecg'])
    696     params['hsel_patch'].set_x(params['t_start'])
    697     callback_close = partial(_close_epochs_event, params=params)

/Users/alex/work/src/mne-python/mne/viz/epochs.py in _prepare_mne_browse_epochs(params, projs, n_channels, n_epochs, scalings, title, picks, order)
    660     inds = np.concatenate(inds).astype(int)
    661     if not len(inds) == len(picks):
--> 662         raise RuntimeError('Some channels not classified. Please'
    663                            ' check your picks')
    664     ch_names = [params['info']['ch_names'][x] for x in inds]

RuntimeError: Some channels not classified. Please check your picks
```

**Comment by agramfort** [comment]
works like a charm.

thanks @jaeilepp  !

**Pull request merged**